manabi/lock.py

```diff
@@ -4,18 +4,33 @@
 import threading
 import traceback
 import weakref
+from abc import ABC, abstractmethod
 from collections.abc import MutableMapping
 from pathlib import Path
+from typing import Callable
 
 import psycopg2.extensions
 from psycopg2 import InterfaceError, OperationalError, connect
+from psycopg2.extensions import connection as PsycopConnection
 from wsgidav.lock_man.lock_storage import LockStorageDict  # type: ignore
 from wsgidav.util import get_module_logger  # type: ignore
 
 _logger = get_module_logger(__name__)
 
 
 class ManabiTimeoutMixin:
+    _max_timeout: int = 0
+
+    @property
+    def max_timeout(self) -> float:
```

Comment:
```suggestion
    def max_timeout(self) -> int:
```

Setter wants int, default is int, so this should probably be int too

Reply:
Thanks for spotting that, its the other way round it should be float. I am passing in float somewhere.